pkg/build/builder/sti.go

```diff
@@ -293,6 +316,11 @@ func (s *S2IBuilder) Build() error {
 		return err
```

Comment:
same here...add the stage info before returning.  I still care how long my postcommithook took to run, even if it failed.